tests/mailheader.js

```diff
@@ -26,62 +26,75 @@ for (let i=0; i<lines.length; i++) {
     lines[i] = lines[i] + '\n';
 }
 
-exports.basic = {
-    parse_basic: test => {
+function _set_up (done) {
+    this.h = new Header()
+    this.h.parse(lines)
+    done();
+}
+
+exports.parse = {
+    setUp: _set_up,
+    'get_decoded': function (test) {
         test.expect(3);
-        const h = new Header();
-        h.parse(lines);
-        test.equal(h.lines().length, 13);
+        test.equal(this.h.lines().length, 13);
         test.equal(
-            h.get_decoded('content-type'),
+            this.h.get_decoded('content-type'),
             'multipart/alternative;   boundary=Apple-Mail-F2C5DAD3-7EB3-409D-9FE0-135C9FD43B69'
         );
-        test.equal(h.get_decoded('fromUTF8'), 'Kohl’s <Kohls@s.kohls.com>');
+        test.equal(this.h.get_decoded('fromUTF8'), 'Kohl’s <Kohls@s.kohls.com>');
         test.done();
     },
-    'content type w/parens': test => {
+    'content type w/parens': function (test) {
```

Comment:
What benefit does this change have?

Reply:
It allows `this` within the setup function to also refer to `this` within the test.

Reply:
Ah scoping, cool no problem.